service/chartconfig/v1/resource/chart/resource_test.go

```diff
@@ -16,20 +16,23 @@ func Test_toChartState(t *testing.T) {
 			input: &ChartState{
 				ChartName:      "test-chart",
 				ChannelName:    "test-channel",
-				ReleaseVersion: "test-release",
+				ReleaseName:    "test-release",
+				ReleaseVersion: "0.1.0",
 			},
 			expectedState: ChartState{
 				ChartName:      "test-chart",
 				ChannelName:    "test-channel",
-				ReleaseVersion: "test-release",
+				ReleaseName:    "test-release",
+				ReleaseVersion: "0.1.0",
 			},
 		},
 		{
 			name: "case 1: wrong type",
 			input: ChartState{
 				ChartName:      "test-chart",
 				ChannelName:    "test-channel",
-				ReleaseVersion: "test-release",
+				ReleaseName:    "test-release",
+				ReleaseVersion: "0.1.0",
```

Comment:
Just some tidying up. We added `ReleaseName` to `ChartState` but didn't update the tests.